sdk/monitor/monitor-query/samples-dev/metricsQuery.ts

```diff
@@ -24,26 +24,33 @@ export async function main() {
   const iterator = metricsQueryClient.listMetricDefinitions(metricsResourceId);
   let result = await iterator.next();
   const firstMetric: MetricDefinition = result.value;
-
+  let secondMetricName: string;
   while (!result.done) {
     console.log(` metricDefinitions - ${result.value.id}, ${result.value.name}`);
+    secondMetricName = result.value.name!;
```

Comment:
Also...this has the potential to return lots of results right? Otherwise you'd exit the loop as soon as you got the 2nd metric. Can you collect all the names to pass to the `query` method?

Reply:
yeah we can collect all the names and pass it to query method. Should I change the sample to that?

Reply:
I am thinking of letting this be - and may be adding a comment that they can pass in all the names to the query method. What do you think? @chradek